### PR TITLE
Add setting to disable the anti-spam

### PR DESCRIPTION
**Pull request by ClearlyClaire**

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

**Comment by Gargron** [comment]
I would prefer if the feature was perfected in such a way that hard-disabling it would not be necessary. Although as I understand it this action is pre-emptive as nobody has complained about too many false positives yet. But either way: This method makes disabling it inaccessible to masto.host (and similar) users.

**Comment by ThibG** [comment]
Good point, going to change that to an admin panel setting.

**Comment by ThibG** [comment]
(It has been changed)

File: config/locales/en.yml [hunk 1]
```diff
@@ -390,6 +391,9 @@ en:
       activity_api_enabled:
         desc_html: Counts of locally posted statuses, active users, and new registrations in weekly buckets
         title: Publish aggregate statistics about user activity
+      antispam_enabled:
+        desc_html: Mastodon can auto-silence and auto-report accoutns based on measures such as detecting accounts who send repeated unsolicited messages. There may be false positives.
```
Comment (Gargron) [inline config/locales/en.yml]: Typo in accounts

File: app/controllers/admin/dashboard_controller.rb [hunk 2]
```diff
@@ -30,6 +30,7 @@ def index
       @trending_hashtags     = TrendingTags.get(7)
       @profile_directory     = Setting.profile_directory
       @timeline_preview      = Setting.timeline_preview
+      @antispam_enabled      = Setting.antispam_enabled
```
Comment (Gargron) [inline app/controllers/admin/dashboard_controller.rb]: If the Ruby class is called "spam_check" would it not be more consistent to name it "spam_check_enabled"? That being said it probably doesn't matter.

**Pull request merged**